src/query/members.ts

```diff
@@ -2,15 +2,19 @@ import { getAllLocalMembers, getUserRoles } from '../store/apis';
 import { INewMembersData, IRolesData } from './types';
 
 export const getChurchMembersData = async (key: string, churchId: number) => {
-  if (churchId) {
-    const { data } = await getAllLocalMembers(churchId);
-    const members: INewMembersData[] = await Promise.all(
-      data.map(async (member: INewMembersData) => {
-        const { data } = await getUserRoles(member.userId);
-        return { ...member, roles: data.map(({ role }: IRolesData) => role.name) };
-      }),
-    );
-    return members;
-  }
-  return [];
+  const { data } = await getAllLocalMembers(churchId);
+  const members: INewMembersData[] = await Promise.all(
+    data.map(async (member: INewMembersData) => {
+      const { data } = await getUserRoles(member.userId);
+      return {
+        ...member,
+        roles: data.map(({ role, roleId }: IRolesData) => ({
+          name: role.name,
+          id: roleId,
+        })),
+      };
+    }),
+  );
+  console.log('members', members);
```

Comment:
Remember to remove the [console log](https://memegenerator.net/img/instances/68620658/a-wild-consolelog-appeared.jpg) if it's not needed.